src/AstClause.h

```diff
@@ -144,15 +144,14 @@ class AstExecutionPlan : public AstNode {
  *       clauses, such as rules, queries and facts. This solution was to quickly
```

Comment:
Can you update the remarks; I feel they are misplaced with the latest refactoring.